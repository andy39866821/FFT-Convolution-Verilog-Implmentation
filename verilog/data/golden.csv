-1583
-1372
-30535
1032
14193
-27689
-2651
17400
16748
-25386
21927
-23333
26190
-16447
18273
9414
28909
20462
14255
-15053
-10634
3640
-8354
-10757
-12581
40560
-9838
8844
13558
-1411
-22601
25001
-11604
-21265
2338
26765
-21895
27423
9919
7091
-7561
12496
-16604
17458
12407
4269
15947
14868
14125
2133
-1806
17743
-7634
4322
-15616
7437
-10312
-12223
22832
17387
4255
-14566
2249
-2593
2455
11037
5027
-29442
-4512
-19100
13917
5832
25381
-11508
17719
-27876
-16419
-5336
18004
-28813
2548
18414
2705
3974
-10336
13404
11843
2852
-28993
22302
33167
-3164
-6928
8379
-28021
13881
3124
2616
-18639
23012
-26394
-21684
8066
24705
-7018
15684
-4927
-11144
17498
-1354
-5646
-3470
34813
11945
-5783
32287
-5230
6706
-4475
12176
-5528
7670
4784
-19636
4615
3001
14395
10912
12559
-9206
2739
-1910
-9269
994
9075
-7915
-666
21274
15138
30645
20112
-3838
-1396
6529
9656
-4280
-12519
17018
13944
372
-29320
6010
-1320
-28303
4895
14431
-5857
-1360
1363
5067
2878
5494
16044
9483
-637
-25991
249
-5187
24197
-3622
16423
-7854
-2402
1990
4588
-29936
-26159
729
21603
-7028
14797
4002
5478
-12362
-18829
-11424
-6723
6206
-30991
11166
7732
-25059
-4529
8330
-1315
10374
-26855
-11435
10806
-4677
625
16222
-14573
-4024
-16732
-459
-16123
-12351
-15362
11779
-2964
763
-6281
30421
-13014
7127
-20929
-4181
16811
15577
-26528
11880
-6702
-27999
-26538
21733
2357
19266
3228
-24787
15009
-17739
33376
-10470
543
4344
108
-7945
-21100
5982
-21261
-9927
42461
-32127
6442
-8360
-23503
-22146
3035
-12794
-23277
13149
4328
21931
1473
20659
22214
2848
-9535
17665
3249
-24188
9081
2270
10661
12137
11653
-11473
-10825
-14796
-10610
23366
-19895
-22977
-4951
3330
-38172
16879
-10351
24771
-22756
-3670
-6248
31584
-6363
4147
-18312
5103
-28337
-472
-8793
9280
10921
-23161
176
-7847
-6105
-19171
3728
-22033
7618
30376
-19287
13231
27654
-3268
9628
-9691
1534
-2186
-14356
-24710
-2442
5128
-4008
2781
3574
-240
-25771
-24008
11213
26504
-7272
-25727
14159
-27139
-22634
4239
-8660
1750
3985
1580
3474
35179
-5789
14122
30289
-9877
-29167
-8245
-19016
27587
20729
-14961
11245
-4138
-7141
-11416
-8316
11423
15583
-14635
-20915
41176
-950
11268
14475
-12281
-5980
-369
15460
-18663
9511
-3571
67
8038
-10584
-115
207
6426
-6913
213
946
11589
-277
-13791
9986
7115
-19129
-8861
-14239
10104
-10344
5468
8442
1299
23145
-24804
2569
-293
-28072
22399
-7306
6265
2267
7377
-42600
4592
17418
10279
7476
5774
14064
-12674
10600
503
33055
-516
15857
-6645
19161
-17583
1014
-35665
15422
-5257
-16113
28733
-25576
13162
-10680
-4168
-8871
23343
-4463
-19101
-7575
20693
-4996
-36979
-4244
26375
-14247
-5997
-6960
11782
26529
4789
35412
-19656
-214
2727
-23023
9282
14184
-1086
-8321
-2789
27683
4681
19996
-23033
-25410
-2743
4242
7135
9481
8455
4681
19457
-19127
12424
-18116
2543
11841
-25144
2122
-6939
-21230
-4653
12211
-3184
8940
400
19839
-4514
32291
-28520
97
9970
-35090
430
-6221
23729
-10507
4140
-26308
29126
-11348
12517
-17908
-9677
2227
-2536
-27098
-7541
2876
-14616
-21738
-17654
3554
-11598
-1960
-11380
15256
-10238
13117
-28248
24006
-13275
-11000
-19400
19328
14723
12322
33625
-7561
-1439
-6749
-34120
9295
6594
-6591
10785
-10878
37795
-1628
-10983
25499
15844
-8129
18898
24530
-15186
-7520
-18268
-16191
18850
4279
-33462
10146
-14836
-21238
-6459
-5165
6992
21811
12555
-28290
27709
-16433
16551
-28495
11225
28775
-12419
5629
3467
11899
4099
25627
-2538
-40674
21635
-15938
-10913
7365
-22464
-11859
-29092
28145
-884
6240
-19962
-7005
-36140
-7165
4886
-24603
19799
31810
-15815
-4120
-8056
6550
-24608
-15015
33913
-20842
-15392
-1036
-31118
956
17110
-18220
-15575
3148
9699
-4600
8863
-346
13604
9003
4320
5123
-21344
-774
-5714
-17649
-12953
2416
28938
-19717
17052
-32931
-1542
3949
-13316
6336
-16408
-181
-1712
15655
-20380
-24270
9683
-3985
35112
2159
1446
-8442
-13690
14417
-25706
-15760
26740
21712
-4817
13367
-12331
-24952
10405
17943
-289
7348
17184
-1449
-14772
2880
32205
85
12360
31117
-16239
6505
9630
9652
-22830
15417
-7282
21829
-15389
-7857
626
5129
30732
17645
6270
14657
10451
8273
-518
-18507
-21538
11424
30247
-26342
31787
-23422
8867
1496
-26572
-13594
4145
14789
-1393
-4335
-25559
4218
3607
10685
15766
19283
7095
10505
-6077
15910
-8754
-2309
-6335
16107
-11083
-13743
-6742
-2967
-19347
31299
6570
-521
-9019
-3389
-16193
8459
-14434
-2275
-20068
22314
-5465
2041
-15601
1875
6245
15294
-15916
-5479
-15527
-11134
-6271
-2127
2338
15667
-26516
1595
17148
20627
7899
-13707
8972
183
36227
-8575
8492
-4856
10961
-12335
-20673
-15975
-5754
-6504
-12497
1327
-15010
5602
5756
8530
-5181
-25083
-2536
7749
-32146
27069
-7261
9629
-10806
-268
1997
-5842
-12446
11301
1436
-1138
-980
38707
-21485
4183
5683
10405
-35068
4645
18651
20240
-9687
-20062
-7030
18756
-9805
26352
25697
-21823
929
4283
1903
31665
-8190
-6079
-6212
-10483
-1318
-12754
3594
-2719
-11677
-10843
12976
1826
17432
4610
5707
12192
9865
4893
8792
-16451
-26836
6829
10720
13278
11046
5442
12204
-15834
1048
22212
3226
8087
-15559
-4326
-109
34356
-10843
-3768
17460
-6321
3490
28578
-10293
19705
10460
-11784
7978
12400
-7110
9099
3134
19055
-5936
-21460
-13505
11470
-21289
-16702
-15173
-9257
-18560
-3274
-30021
5314
14908
413
-24793
-7060
31722
16874
-14528
-20966
-10238
-15634
8726
18149
-10083
5987
26476
-6821
9298
18124
-14156
-1564
-19231
-6036
589
15890
-3323
-15710
16870
-6834
5826
-13197
-8904
25898
-10123
22967
-22346
36492
-3478
-7385
26420
2043
-41291
4015
7753
-8908
191
-19035
7337
12601
-30582
-1549
28362
-16472
-5559
4356
8924
12533
-47
9062
2213
2133
2524
19197
-25585
-10898
-17215
2848
547
-1849
604
18582
4994
-4690
37649
-23436
7924
32229
20054
-35278
2400
7852
17174
11628
2759
-3427
4280
12891
101
3922
29698
-23760
7076
-20981
-21708
14306
-20702
21517
-32293
3541
11564
-12364
9554
1279
-88
-4624
-7152
13249
20596
-5863
-8026
15862
17524
-19740
122
-6035
-2236
-43259
-855
2786
12150
-4890
1012
-7684
7487
-13135
-2940
-3056
12111
-1843
181
15765
-28994
7359
8484
15079
12031
-3380
1515
-1588
-1618
-27145
32587
-21918
11286
-10493
-20297
-2115
24166
3436
-22197
-12145
9855
-10531
14822
-11791
-10117
3818
-8389
41551
-25382
11377
5763
-20250
27830
-30986
23479
-737
2662
-27809
-16493
-1769
-8353
4458
106
-14091
-8033
2678
15536
17472
-319
-17575
-14563
21593
20028
-1358
27262
-2791
-829
-4561
1612
-20567
1087
14201
-26959
-15195
-22920
22842
-2848
9151
8574
19629
-4069
-11899
383
-8186
36261
10233
12312
-1863
-5905
14572
-11614
16173
-9608
-8099
3315
-9556
3369
-12194
15988
8480
-27303
9553
-29887
18557
13026
-11107
35456
2437
9020
14981
20708
-25573
-10654
10010
-15801
-20780
-26252
-4311
-36442
28556
28855
4356
13305
3354
-8929
727
774
-16968
9215
21404
13869
-6043
-1546
14921
22354
15192
-17985
23916
-13003
-14100
-20212
-14894
-18186
5962
-20901
-9038
-7874
-17252
336
-2049
10267
8937
-8305
11262
-20270
11194
2216
-4397
-7362
-12244
27420
-26031
847
8190
1992
30605
-13800
7796
24858
-6858
-20407
17128
28798
-3267
36173
16671
-5408
2247
1441
23486
-19511
2786
-33731
-8448
6440
6537
-9859
25473
22558
409
4584
8838
-24858
23979
8435
836
23942
-12663
6657
38292
-6490
8106
17866
17394
5964
-20079
9038
27514
3259
9736
1984
-14652
6140
9632
9405
13467
-24306
20683
-6572
4999
-1985
-5626
6440
-5394
-19792
-2198
15549
15376
1484
-4554
-19768
-5818
2001
3066
-5229
1344
8840
-15050
-10870
24825
7493
-7012
-278
45366
-12628
12999
-12049
-1851
-12493
5406
11237
-10927
1063
15211
-36080
-14883
-11327
17473
1720
-9361
14802
-4267
-19374
2199
-2009
3025
27508
-4152
9365
21610
-15241
-7619
11831
-13682
-19406
-8529
18815
-1320
-25925
17952
6088
1334
4152
-13785
-16754
33963
-8704
20671
-17784
-4155
9565
2959
-14043
7832
2413
11264
-19605
-4686
20228
-1988
-6681
8887
27481
-7074
-19947
2885
-13264
441
11803
11082
-22320
5200
17264
-11347
7816
-3300
-15458
3943
10049
1976
-4655
-4925
13729
-15637
1842
1882
-2155
-10059
17371
-12448
-1240
-3853
6268
15569
-11800
-4287
19898
4770
-22255
-3640
3354
8914
-17664
-24021
12426
-12580
17891
13996
-30231
23377
-32857
5237
-10027
-14370
-5466
15260
-16415
9704
-5762
-13859
-23178
-14847
-17328
-4526
-4216
-7294
29182
-3735
28144
-5622
6401
11491
-8668
-2328
1664
-20587
-16374
6655
1881
22242
24202
-9774
19269
13839
-7766
-23767
27650
22835
-423
-18064
-3784
-12823
-11680
-2917
-9589
-23534
-20922
-21244
27981
27554
-11862
-3747
17659
21993
-5692
34650
12566
-16764
35971
-5279
684
-3330
-9308
7242
-10330
14463
2978
-17449
8603
13429
8794
30697
10025
9028
19503
9428
-34191
-22470
-2103
-16308
15912
-13645
7434
-7546
15385
1250
-10659
-17024
-3319
-948
1054
13387
-10084
-8418
10759
-4948
6780
-19664
19892
4339
-4511
6561
2313
-39295
28953
20556
-16973
-7396
27373
6270
2494
-4861
-3134
-17118
-21118
-27118
8278
-5284
616
12769
6262
-30974
-5170
-26585
-15325
-360
22081
1220
-1865
-22658
-21713
-28363
-2361
-13202
-11922
-16609
21215
5434
-193
-4293
19413
9636
5424
-17386
27045
-6736
-7121
2054
3763
5312
718
-5409
21315
-5119
-13672
18470
-12202
13196
-2781
682
-400
7063
11376
8095
6558
-19396
5681
-7761
-1327
-14609
-1042
17829
27725
4633
-28630
-7489
13087
-5435
22192
4185
-5652
8944
12518
-6869
-1123
-13932
-10272
-18079
-9605
10720
1181
4414
33551
-12174
-3203
-14743
11295
2572
-5218
4307
14090
-9617
-11379
3345
38790
-174
6482
12579
-27390
-1130
6063
10571
-3700
6218
21891
-6104
-15699
22083
11000
-2648
-14392
40994
-24314
-14566
-3820
17143
-7920
-2506
-11590
4943
13265
-10975
1335
-11182
16244
-12314
239
-13461
22538
14001
10585
-8345
-6102
-1776
-4876
-5758
3062
-6591
17578
16640
3667
13699
-23338
-336
13978
-5804
-10754
-24519
10391
3225
-15581
6720
-18997
-6662
-4663
-17945
-25541
2144
9578
-33
-7011
4631
-15462
4408
-17567
1303
2238
-10532
8497
19815
13603
-20595
370
12880
-765
-10657
-10584
3737
-5101
-12667
-27540
32268
-14689
-9728
7497
-11112
-10448
-36667
-10452
-4392
18399
-29165
23144
4141
-4660
11382
-12215
-4732
-2647
-220
-3747
-3866
6197
15958
-37906
11492
-3056
-18573
15045
22803
18929
-7712
8408
28575
-14417
1386
-740
12936
-17015
3051
-10303
15964
20797
9514
2161
6354
-6254
12628
-46007
-995
7522
7068
1163
-31810
16447
-32274
4703
-9122
11943
-16465
14792
20782
257
21653
-6598
10152
6775
2890
5250
17191
-6368
18232
597
-6127
-20045
10927
19508
-3712
-20222
-4514
-17210
-8464
-880
4090
-549
-1121
-17134
8116
6371
-9780
940
5461
10422
-18099
-4490
-10127
136
9289
-22059
7342
4952
-20626
-8632
16377
-39329
-7549
9988
-14810
17405
24501
-7425
31361
7269
-12623
-5056
577
35739
1399
-4363
848
-8811
-14004
-3374
-23095
3822
7175
-24164
8985
1736
9865
11069
32109
-22642
2738
-28885
-10005
-7254
-4737
7118
-6679
8242
17034
-15007
8934
-20619
19611
1331
-3804
-17783
16686
28106
14014
-11267
13871
-9512
-3208
-1591
2029
-5702
1251
10285
4991
-125
-6903
20400
17059
-4248
14961
-16822
-11868
569
-5854
16718
20583
112
-11565
3412
-4243
-9682
8900
-7277
-12457
-12490
-5476
17294
4114
11363
239
1421
12265
-3529
-1537
-11922
3261
-5560
-4788
3640
-6868
30310
-2780
546
-39834
25834
15411
-12350
21024
-2381
1931
11707
-3701
-24860
20958
-17726
14462
4480
-21444
-4223
3487
-34537
27714
399
4066
-25686
12574
-6316
19433
1982
-13255
-5219
1018
-5438
28252
9049
-4849
-1407
-4819
-27937
-15404
21591
15219
28123
2301
-28018
-139
4985
-9786
-18456
18438
1094
-3291
6792
23654
4975
1864
-709
2222
4118
-22506
-2057
-6660
20432
-15051
8154
-16561
5730
233
-15189
1970
-21416
-2864
-742
6929
-10992
28633
-2831
8827
7500
11934
-18939
14912
-12445
-21662
3375
4524
-19975
-2647
5523
12914
25340
7438
-9266
-11929
-24310
8744
5016
-5325
22412
2685
-5954
-16343
-4227
5838
13488
-33141
1400
15800
-26978
16885
16098
-3625
-3646
-2441
-409
-5972
1731
-7330
29882
-16044
15930
2189
7970
8515
-5536
1747
7768
4019
-2795
-1448
24285
19827
9906
21809
-17329
-4551
5200
6724
-24026
-17166
-860
12421
-11215
20314
16220
2903
9778
-23
-1140
13538
23127
-21860
7331
-13918
12457
22346
12802
9238
-28524
-15278
-17656
-8775
19566
-7057
14870
12034
-16420
-21658
-77
3301
-12118
17913
-4465
17970
17769
-8772
-12155
1979
-8646
-25557
25547
-360
26286
-39162
23381
-17701
15680
-2964
2312
616
2262
-3876
-23701
-29561
23218
-3643
-18858
-13370
29759
25924
-1277
-9392
-11705
6701
-10733
-7444
13999
-3227
20507
-9127
3572
25343
18898
-27619
12124
-20637
-17521
-12909
17866
11425
-2542
9650
4893
-7091
-15920
1995
10086
2824
-15778
2733
12696
-33242
-15948
11543
-27679
-6769
-5125
11032
492
-727
21455
815
-11438
-7962
-13185
3965
17458
-15310
37365
14667
-2446
3076
19471
37705
10896
-22066
14561
9167
6041
-16241
1106
-1293
-960
31305
-6678
22024
-24494
-1182
5838
-12507
17794
-26848
-803
3438
3457
-34710
37643
-16044
-290
14391
10027
-13888
-15711
8660
31201
11969
-15504
21918
-9227
-11066
-4244
-19726
-6860
-6561
23044
-11196
-23599
17857
-36572
-4484
-24227
9028
301
18165
16362
-1270
-26168
19810
-24392
-162
19124
-4252
-3776
5094
-7738
27006
-18966
794
13052
7402
16431
-732
22048
-32809
-28032
3294
7708
43047
-3598
-155
27995
-11429
15698
-20788
38822
-15490
6238
-26080
33577
-25293
-7635
-20113
18005
-4649
13766
811
3526
-25921
-24919
-4287
-12492
9162
-10376
6693
32205
10681
6798
-9922
18412
5275
-4651
31857
-25625
19752
-2649
-16587
-17265
1637
-312
19005
-23229
1489
12330
-22960
7636
-404
-24053
9123
-6594
15565
-13856
-18805
-22422
-16721
18656
4496
25142
-2882
5012
-19731
-2770
8848
-15158
-4491
3607
-35609
25771
-22848
11737
-16796
24840
-19019
2995
8714
-3482
-7787
28706
-20396
5251
12201
3248
-3903
-37979
-10325
-8081
6871
-12967
22583
-1222
-23094
20149
-14281
-25150
3678
-21913
27093
4420
3119
5082
-9355
-17854
14697
-8200
6978
-19984
-10953
20361
22214
-1540
4807
10133
32652
28509
11237
3037
8907
-20114
-26403
-10551
401
24536
-10100
-15476
19044
-6685
31378
-256
-16451
3919
7853
-25936
9879
4940
-19373
521
6408
11210
8693
-5850
-25075
-1515
18923
17714
20085
2528
-14240
275
-9571
-1064
100
-20839
4214
11058
-8215
-3675
32083
5321
-11551
-4353
9094
-9725
333
-2669
-7254
-2288
-5697
-16618
-19665
2469
7055
-17161
-7317
-19002
-37441
5938
-21604
-17051
22011
1738
15012
-5694
17644
2533
6200
-4224
8266
-25940
19774
-14326
-26376
29464
-135
-21832
6604
-15937
7939
6906
-1391
314
-7810
-495
13402
-14942
-21747
-13624
16074
5462
4352
26776
-10680
-6709
-1602
-35477
2109
-1862
-12625
-3492
-9501
-9479
2202
5507
-16778
-9791
-6774
5277
14745
-12527
-10303
3702
6058
18728
-4392
-9783
7490
-11818
-18936
18333
-65
5703
4530
-14120
-920
1119
-9539
-33030
-1874
-2421
20101
533
-10977
-7224
-5122
20659
-14187
4173
-6341
12122
21339
-11535
-15794
-29204
-13411
-8110
-4143
1390
14399
15799
-23625
4044
-21522
17138
7960
-18214
-12014
-1700
2
-38530
7072
14492
-2829
-8202
-10980
1571
-25646
-11243
-3513
-6935
8734
4520
-9335
22558
25183
19880
-6769
-12210
-16488
-4395
3324
14510
-20287
7737
4093
-25623
-3043
1555
9501
2417
-14555
-25543
-4893
4423
-223
-9668
16503
16530
17071
-18028
15179
27938
-421
12773
5289
17114
19957
14202
-17300
-9966
19996
7974
-7772
24924
-25014
-12651
5129
-13532
-14226
22382
445
4601
36780
-5010
4338
26810
8295
19703
-3770
8349
16066
-6115
-9224
-27577
-11399
19098
-25781
-10381
-8907
13759
-23273
11084
-434
30712
-16198
24656
16469
21009
4409
-33004
12752
1351
15139
-1865
-7788
-8557
-14572
37
-19677
-19386
13188
7110
27187
-2179
-14478
375
4264
5818
-4713
27940
-17592
-8622
14773
-18248
25173
15625
-24635
10196
-2336
-34910
-17251
16296
-27737
-4928
801
-25508
15201
-1011
-22255
-3780
-33642
14873
-1663
18869
10079
-10608
-11733
5119
4244
-2907
-18915
29638
-12507
23822
-11633
-12907
-3107
928
-39668
16202
-12765
-9130
-11592
-279
-961
-3252
-4663
26587
-894
9406
-9434
-10100
-8391
-4795
2309
-9808
12043
-16309
4187
17079
2825
-10290
3931
21677
-2056
-9382
7153
2650
-24929
33801
-4899
-9968
24908
27190
-7657
9689
2414
-28939
28655
5482
-3224
-2233
-23131
-7398
5186
10721
-8875
13047
9246
-9375
18575
-19650
18527
6930
-26262
41828
-4374
-20671
13847
3084
-14394
6483
-6896
29843
-18526
19674
-22324
6025
2222
13215
13632
9903
-23282
1814
2038
9359
-25644
-10623
1921
-10229
3360
7227
27344
-27851
-14786
-4871
-11930
14232
-4723
-5584
-26694
39275
-35700
21856
-15874
-3472
-17728
11928
18110
-21617
21318
-3413
-5240
567
4750
12629
-17967
-18590
7116
-461
-17607
14499
4119
-10301
27565
-16685
-22089
10349
-8914
-8377
-5626
9459
14413
-17163
-7670
34115
9554
14942
-35570
-165
-6095
-5676
-10668
279
9573
18732
-6513
-3930
-7863
-4714
-28194
8473
99
-736
16576
-10968
41440
-16797
12370
-1494
11958
-16248
-23286
19305
-25034
22458
-18537
706
-16424
-6652
6494
-16051
10753
-27093
22897
-21711
8336
6451
9387
2744
-23098
11272
6891
1845
-20070
6720
-10925
-8995
2674
6438
-7268
17448
-10342
4301
17957
1353
18953
-18402
742
26135
-2990
35077
4509
11742
-11668
-21009
-39817
12199
15737
25573
4143
-2057
-11549
7137
-10757
9470
18135
-28200
32631
-26519
11220
-18469
7919
-11396
12413
-8505
-35345
10222
1350
32630
-15295
32742
6559
5082
-9311
-16908
10425
-12644
-3218
-32507
12946
1134
-11446
1980
3420
156
-7069
12566
-11791
10599
5075
7263
-25778
7280
-1901
114
-17928
8289
18854
-42099
-17838
-9277
-6082
10530
-40705
23205
25115
23266
238
3302
-2536
26540
7011
6618
-13665
29948
-2036
-39
7409
-14454
6305
853
21033
8040
-11458
-9137
-9599
12548
5340
-8072
3041
-2964
-2947
19934
9611
10197
-15217
26567
22702
-10803
-774
-3119
36742
-10692
17167
17819
-4190
16141
-14746
7462
-15662
-265
6721
23840
-3240
-14516
4869
23640
-19193
-5024
-10198
10852
37997
-1124
1079
17447
-29770
-18392
-31567
-7793
-3536
11725
6833
-23031
-751
21613
-6726
3431
5568
1741
5621
9622
-9864
8059
-25178
30036
1291
-8649
-40034
3079
20514
1550
822
-10038
1319
11707
-12513
-33192
-3058
-10317
-4756
-2077
-25331
-11726
-310
-8094
2509
-4714
-8145
565
16700
18684
-24274
10946
8296
-9174
-3183
-16108
7914
3558
5847
-9592
-34414
7643
-8314
24615
4877
16060
12724
24508
-6048
-10463
1765
7416
21282
-28496
12567
-17882
-7213
34433
-7880
7331
-2021
-9553
-6181
42022
-29846
17043
-6711
-2456
-16059
-4545
-18469
27873
-5506
14816
17810
1403
-12154
8098
18000
-5483
-15373
-14077
8252
19257
-6683
-8520
25798
-9624
-10566
-6995
5150
25487
-21082
5770
16921
-14651
3101
155
21505
-29376
-11850
936
-2166
27564
-30541
-27046
-20293
-16021
-7389
13315
-20284
10416
-6451
-21239
-3582
-34423
19482
-22684
-17140
-5266
5027
2360
8414
-16351
-18429
20649
2202
19099
5277
8289
-2173
26015
-1687
11507
7760
9396
11949
-13455
-35242
9351
6680
-23463
-3623
6036
7021
20297
-15851
967
15524
-4408
454
10426
13924
8065
-637
-666
-21629
-29010
-10788
6142
15379
-9938
23564
6358
22027
15227
-25441
27809
6281
-755
5224
-7850
-14061
5173
9484
29439
14948
-999
19465
-24040
-2122
-1462
17239
-1421
34065
-5599
-3519
-23714
-7011
-10151
-6168
-28419
-20214
1264
-12447
-6772
-14071
-8991
4985
9114
3522
36855
-1948
-5653
8300
10987
1059
12489
-2793
9772
6462
6613
-11342
5507
-1750
451
21391
7280
-17372
-8859
11936
-6064
-12214
-20616
-9586
-5233
14900
3434
-23071
-1412
-12042
7990
6307
-24272
-1791
-15907
-8144
-4810
12675
8298
7961
10298
-5149
22410
14279
-9880
23838
-22598
16064
-17540
24008
4767
5635
-342
10252
-5873
11159
24596
22102
-8311
-28851
-21302
-16953
17019
-28744
-26294
25288
5671
6994
-10421
12052
3393
-9678
7923
-20360
25771
-27661
-30504
7111
10033
14641
18138
459
26018
22522
1475
11078
-10051
-11113
-10916
-4225
-6864
-269
14675
-10156
-13149
11731
4986
8387
-7213
-3843
16701
-35457
-21372
-19342
11434
-9900
4191
1803
-32779
4112
-7566
8250
-20427
14587
-1886
1846
-6062
13677
19181
14072
-13214
4845
1384
-6470
17116
6156
-5264
21997
2669
-4752
-7581
9639
6358
37201
-24054
-2983
11039
6874
-1663
-26769
-21641
8116
23558
21233
-1765
-28379
-2601
15845
19416
-13020
-17752
2580
11250
10262
-8416
-23994
32637
-19683
7024
529
4350
18839
3489
-5718
30705
-16398
19138
-24970
17929
232
-19718
-6435
17668
11288
-11427
12566
-20858
-14393
5748
2878
-4240
-19129
-5680
-15880
-13029
-5258
-18554
-21170
-7643
11889
2340
-19606
12627
658
-17743
-3613
-31207
8858
-24981
4209
18569
13481
4978
-11788
221
19512
6900
-12506
-12663
-6562
16875
16865
-17460
-11070
-4084
17558
12880
-9742
10746
-10671
-1271
-11580
-2582
1487
-15006
-33763
8811
4139
5391
510
16869
414
8391
16376
8881
18889
-5103
-15489
13554
10885
19541
-4545
-598
5022
-2787
11029
15676
4457
19928
408
-796
10658
-14431
-23261
-5660
5072
-8521
-2259
-6283
9847
-28362
-10569
-18719
-13512
4435
-19249
-10944
-2664
-11910
-14738
-2491
-11260
-5687
3913
-5631
16942
-11891
2828
29200
9276
112
5017
11117
12610
-22713
11223
14158
7653
-16326
-25510
-8457
5235
-15780
-1986
1674
-13168
-10673
17331
-21800
-18301
-2491
1173
3914
-995
-10015
-81
-8591
-10893
-20776
-2850
6721
-9730
39815
6812
15875
20918
5011
5649
-28303
7965
15296
144
-13832
16979
-8692
-4662
-14010
1453
-12594
1737
4660
-7708
18373
-17943
-7744
-5931
-10663
3706
8954
23875
-30785
2834
11768
-10264
21812
17971
-6210
-1948
4257
20296
28187
-8163
31724
-4030
460
22528
-1014
8739
4363
1864
-18657
25261
16372
15192
-6337
-8708
21515
-9346
-8092
-10096
2357
1976
-7885
-21620
-18583
-24481
3695
-15518
-39935
13986
12241
18294
-3704
-4557
-4990
2257
-12358
18995
4974
-12856
16107
-13265
-24975
13031
10487
-16160
10977
-13487
-29852
-7866
-5180
20902
3103
1778
-9931
3082
25057
-21673
-22245
9056
-9690
-39567
-12146
-12405
5887
-36082
9075
-417
-28916
29687
11363
-8295
-11127
11634
-16531
-4878
-7691
8983
-3661
4081
7102
-1411
23934
-1714
3571
1203
5524
7024
18113
-8417
-11705
20363
1870
-6185
-27377
15614
7376
-7943
-14081
23847
4195
1893
11996
-9966
-1665
14137
19033
-3655
28902
2563
6742
5614
2636
-27348
6983
-7171
14957
2067
28172
-22847
2770
-5852
10972
20124
13699
18663
-8418
19891
-11025
7779
-21846
-2472
7056
-11767
-2248
3877
-2661
-21778
-8780
17634
25460
-30420
-21359
-1618
34357
2590
12143
-19719
20169
-23845
19525
-22633
-8594
33788
7400
-21795
-19361
17271
7338
-11861
-3524
-4388
-22073
4962
23918
-18453
-285
-11808
-22894
689
16060
-3840
9599
-16899
15939
2422
27968
522
8099
-9452
8864
-11860
12186
-31825
-15963
-11054
5363
26043
-24630
3195
-12416
23491
-26354
15415
11684
-9985
-8938
1946
-20267
-5712
-12248
-13175
6973
3127
22141
3034
-8857
-23129
20452
-17336
15639
-3583
15754
-9593
14203
-8601
5694
-5971
-9495
18996
-21166
1287
-16134
-156
5466
8156
-13851
-23613
38067
-12381
-3423
-39093
-7770
-6564
28468
-186
-23206
-455
-309
161
37766
-4592
11817
9654
14647
9515
4319
-18241
-5422
11578
3414
-11421
-24793
11563
-17979
22554
8495
-9476
17127
8555
1806
-7624
-3202
-7398
-22194
-23347
6747
-3899
-3750
-18753
12199
-12471
3675
8660
7460
-31623
20465
-4716
13206
3479
2529
-7062
5570
26721
-26345
3584
-2207
18207
-23977
7378
31479
-11623
386
-5302
37695
-15878
18512
-14040
-17701
21735
-21001
-8402
-14636
-13177
-2858
-14667
20841
5936
-26520
-6771
20637
-22564
-21286
-15081
12404
-23986
10690
-5508
-24458
-14627
-6450
-24520
9949
9070
6143
-21145
-7318
36393
9194
-22976
-13052
-22910
4701
-10873
28033
-2782
10171
-3556
-14200
-19035
4939
2977
6956
3472
-11391
1283
-9842
26755
1678
24721
-29333
3051
19376
9989
5023
4741
26108
-1206
8257
-17704
-13682
25541
8300
1416
14913
8263
8100
-16006
13705
9418
-3849
-21553
14159
-2027
6920
-10369
7057
-1482
21110
-7127
6667
-14269
13920
-18904
20599
1637
-13400
16090
-5514
