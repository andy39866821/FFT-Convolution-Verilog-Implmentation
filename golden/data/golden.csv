-20506
-11590
15533
17476
31623
-16181
-31872
-6335
4964
8857
-361
15759
3979
22912
-11183
-4106
-24070
10338
-18728
18157
-19462
2603
-9667
-2063
1238
9219
-8585
-10772
6799
22526
-10865
6249
3994
7183
20276
2825
